mypy/test/testextensions.py

```diff
@@ -0,0 +1,115 @@
+import sys
+import pickle
+import typing
+try:
+    import collections.abc as collections_abc
+except ImportError:
+    import collections as collections_abc  # type: ignore # PY32 and earlier
+from unittest import TestCase, main, skipUnless
+from extensions.mypy_extensions import TypedDict
```

Comment:
This line should somehow be `from mypy_extensions import TypedDict`, see my comment on `__init__.py`.